### PR TITLE
Improve error message in OrbitServiceInstance

### PR DESCRIPTION
**Pull request by antonrohr**

### CODE REVIEW

File: src/SessionSetup/OrbitServiceInstance.cpp [hunk 1]
```diff
@@ -82,7 +82,8 @@ ErrorMessageOr<void> OrbitServiceInstanceImpl::Shutdown() {
 
   bool wait_finished_result = process_.waitForFinished(kWaitTimeInMilliSeconds);
   if (!wait_finished_result) {
-    return ErrorMessage{"The OrbitService shutdown attempt failed."};
+    return ErrorMessage{absl::StrFormat("Shutting down OrbitService timed out after %dms.",
```
Comment (beckerhe) [inline src/SessionSetup/OrbitServiceInstance.cpp:85]: Nit: I would prefer a space between the number and the unit.

**Review by beckerhe — APPROVED** [review]
Thanks!

File: src/SessionSetup/OrbitServiceInstance.cpp (hunk 1 above)
Comment (beckerhe) [inline src/SessionSetup/OrbitServiceInstance.cpp:85]:
```suggestion
    return ErrorMessage{absl::StrFormat("Shutting down OrbitService timed out after %d ms.",
```

Nit: I would prefer a space between the number and the unit.

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**